src/services/walletconnect/__tests__/WalletConnectWallet.test.ts

```diff
@@ -344,36 +344,36 @@ describe('WalletConnectWallet', () => {
       expect(emitSessionEventSpy).toHaveBeenCalledTimes(2)
     })
 
-    it('should call emitSessionEvent with the correct parameters', async () => {
+    it.only('should call emitSessionEvent with the correct parameters', async () => {
```

Comment:
Don't forget the only.

Reply:
Addressed in 09db2d41623fff256ef808f743ae3ee7525091b0.